docker/Dockerfile

```diff
@@ -18,7 +18,7 @@
 # >>> docker rmi $(docker images -q) --force
 ##############################################################################
 
-FROM continuumio/miniconda3:23.3.1-0-alpine as build
+FROM continuumio/miniconda3:23.10.0-1 as build
```

Comment:
Do we need to pin the version number that precisely? Can we just use `latest` here?

Reply:
Hi @gboeing, that's a good question -- perhaps we can.  From memory, the reason I did a specific version at the time was, that was the latest release that supported Arm64.  Previously, I had used the 'alpine' image, which is smaller size, but doesn't reliably support arm64 (mostly only amd64).  Perhaps that is why the alpine image is half the size though.  If you believe the 'latest' miniconda3 reliably supports arm64, and using 'latest' is unlikely to introduce incompatibilities, I can't see why not (they just released an updated version 3 days ago, so already advanced since pull request lodged). 

I'll look to updating this after I hear back comments from others later in the week, in case there are other things to shift.